packages/client/src/utils.ts

```diff
@@ -114,13 +120,105 @@ export function computeRatio(
         break;
       }
     }
-    let val: number;
+    let quotientValue: number;
     if (denom[j].value == 0) {
-      val = 0;
+      quotientValue = 0;
     } else {
-      val = num[i].value / denom[j].value / scaling;
+      quotientValue = num[i].value / denom[j].value / scaling;
     }
-    result.push({ date: num[i].date, value: val });
+    result.push({
+      ...denom[j],
+      quotientValue,
+    });
   }
   return result;
 }
+
+/**
+ * Flattens a nested JavaScript object to a single level if key/values.
```

Comment:
did not understand "to a single level if key/values." Please clarify.

Reply:
Typo in my part-fixed